### PR TITLE
Shaker sort in Python

### LINKED ISSUE #2155: Shaker Sort
- [x] C
- [x] C++
- [x] C#
- [ ] CoffeeScript
- [ ] Dart
- [ ] Go
- [x] Java
- [x] JS
- [ ] Kotlin
- [ ] Php
- [ ] Python - @chaithanyarlk
- [x] Readme
- [x] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by chaithanyarlk**
Fixes #2155

#### Checklist:
- [✓ ] 4 space indentation.
- [✓ ] Coding conventions are followed.
- [✓ ] Input is taken dynamically.
- [✓ ] Sample Input / Output is added at the end of file.
- [✓ ] Logic Documentation (Comments).
- [✓ ] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- File has been added 
- 

#### Languages Used:
- Python
- 

#### Files Added:
- Shaker_Sort.py
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by chaithanyarlk** [comment]
I have created a pull request again as I was not able to squash .So I cloned it again

**Comment by chaithanyarlk** [comment]
Can you ignore the second commit as I made a wrong it is of other issue which added on same branch
d0ece70 this the commit hash

**Comment by asha15** [comment]
please include the issue number in PR template

**Comment by chaithanyarlk** [comment]
Please review it I have added the PR template

File: Intro_Sort/Intro_Sort.java [hunk 1]
```diff
@@ -0,0 +1,205 @@
+// Java implementation of IntroSort
```
Comment (asha15) [inline Intro_Sort/Intro_Sort.java:1]: please remove this file

**Comment by chaithanyarlk** [comment]
Please review it I have removed the file

**Comment by asha15** [comment]
oh You have deleted so many files :( could you create a new PR?

**Pull request closed without merging**